### PR TITLE
Agathemerus CTS files

### PR DESCRIPTION
**Pull request by ChiaraPalladino**
Here are the two cts files.

### CODE REVIEW

File: data/tlg0090/tlg001/__cts__.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0090.tlg001" xml:lang="grc" groupUrn="urn:cts:greekLit:tlg0090">
+    <ti:title xml:lang="grc">Geographiae informatio</ti:title>
+    <ti:translation xml:lang="la" urn="urn:cts:greekLit:tlg0090.tlg001.opp-lat1" workUrn="urn:cts:greekLit:tlg0090.tlg001">
```
Comment (PonteIneptique) [inline data/tlg0090/tlg001/__cts__.xml]: This should be `xml:lang="lat"` here

File: data/tlg0090/tlg001/tlg0090.tlg001.opp-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,451 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"
+schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+<?xml-stylesheet type="text/css" href="../../xslt/muller-xslt.xsl"?>
```
Comment (PonteIneptique) [inline data/tlg0090/tlg001/tlg0090.tlg001.opp-grc1.xml]: Please remove this reference to a local file.

File: data/tlg0090/tlg001/tlg0090.tlg001.opp-grc1.xml [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+                    </div>
+                </div>
+
+               
+                <div type="textpart" subtype="chapter" n="3">
+                    <div type="textpart" subtype="section" n="8">
+                         <head>[Περίπλους θαλασσῶν]                   
+</head>
+                        <p> 
+                            Ἔστι δὲ ἡ μεγάλη Σύρτις σταδίων &#885;ε, ἡ δὲ μικρὰ σταδίων &#885;αχ&#884;, τὸ δὲ
```
Comment (PonteIneptique) [inline data/tlg0090/tlg001/tlg0090.tlg001.opp-grc1.xml]: Avoid use of xml entities : http://unicode.org/cldr/utility/character.jsp?a=0375

File: data/tlg0090/tlg001/tlg0090.tlg001.opp-lat1.xml [hunk 4]
```diff
@@ -0,0 +1,447 @@
...
+                <head>
+                    <title type="main">AGATHEMERI ORTHONIS FILII GEOGRAPHIAE INFORMATIO.</title>
+                </head>
+
+
+                <div type="textpart" subtype="chapter" n="1">
+                    <div type="textpart" subtype="section" n="1">
+                        <head>[De veterum geographia.]</head>
+                        <p> Anaximander Milesius, Thaletis auditor, primus terram habitatam in
+                            tabula describere ausus est; post quem Hecat&#230;us Milesius, mult&#230; vir
```
Comment (PonteIneptique) [inline data/tlg0090/tlg001/tlg0090.tlg001.opp-lat1.xml]: Avoid use of entities.

File: data/tlg0090/tlg001/tlg0090.tlg001.opp-lat1.xml [hunk 5]
```diff
@@ -0,0 +1,447 @@
...
+        </profileDesc>
+
+        <revisionDesc>
+            <change who="CP" when="2016-03-03">Created the XML file and made the file CTS compliant</change>
+        </revisionDesc>
+    </teiHeader>
+
+    <text>
+        <body>
+            <div type="translation" n="urn:cts:greekLit:tlg0090.tlg001.opp-lat1" xml:lang="la">
```
Comment (PonteIneptique) [inline data/tlg0090/tlg001/tlg0090.tlg001.opp-lat1.xml]: `xml:lang='la'` should be `xml:lang='lat'`

File: data/tlg0090/tlg001/tlg0090.tlg001.opp-lat1.xml [hunk 6]
```diff
@@ -0,0 +1,447 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"
+schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+<?xml-stylesheet type="text/css" href="../../xslt/muller-xslt.xsl"?>
```
Comment (PonteIneptique) [inline data/tlg0090/tlg001/tlg0090.tlg001.opp-lat1.xml]: Remove reference to local file

**Comment by PonteIneptique** [comment]
@ChiaraPalladino > I have reviewed the source. I would really much prefer html/xml entities to be converted to unicode, such as `&#339` into `œ`. This makes the source much more compliant with all kind of efforts (and not web only)

@lcerrato > Except the style issue, the file seems good to go. Because this is not a conversion, I would like to get to approval before I merge when things will be corrected.

**Comment by lcerrato** [comment]
Hi, I'm confused a bit about the placement of the work. 

We haven't been including OPP texts in the Perseus repos, so wouldn't this be better named with a perseus identifier? Or should this be part of another collection? I think there were earlier discussions about this?

I don't know for sure as this is an area of the workflow we are still defining.

**Comment by AlisonBabeu** [comment]
I'm not entirely sure if this is relevant to this thread or not but for all of the Agathemerus editions recently ingested into the catalog, since I didn't assign them URNs, and didn't realize that they were in the Perseus repo, all of the editions have OPP urns....

**Comment by ChiaraPalladino** [comment]
Hi all, I don't remember how we came to the conclusion of naming the file this way. As for me, I just care about assigning it its own urn and having the text CTS compatible, so it doesn't matter how it is named and I don't care about having it in a special collection. Since Alison has already OPP urns maybe it's better to leave it this way, but I don't know if there is a relevant difference. 

@PonteIneptique Thanks for the corrections, I'll take care of them asap.

**Comment by PonteIneptique** [comment]
Actually @lcerrato , we have opp for Latin on stoa0275/6. It certainly would not be the first time. Which does not resolve the question at all :)

**Comment by lcerrato** [comment]
Looks like those Tertullian files were added before we discussed the new workflow. (I would also note there is no header info there, no logs, etc....).

It wasn't immediately clear to me what was going on and the identifier added to the confusion. I thought this was an OPP work that needed review on the Leipzig side.

I don't want to fail to address something in a timely way and therefore hold up further research in a situation like this because of competing priorities. Perhaps a label that better fits the needs of this type of situation?

**Comment by ChiaraPalladino** [comment]
I have no problem in changing the title. I could put my initials in it, like cp-grc1.xml and so on. I'm concerned about keeping the files that @AlisonBabeu just put into the catalog as OPP, also because I planned to provide the corresponding text of some of them.

**Comment by lcerrato** [comment]
@ChiaraPalladino no need to change the URN or title.

File: data/tlg0090/tlg001/__cts__.xml [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+        <ti:label xml:lang="eng">
```
Comment (lcerrato) [inline data/tlg0090/tlg001/__cts__.xml]: eng --> lat

File: data/tlg0090/tlg001/__cts__.xml [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+            Geographiae informatio, Geographi graeci minores Volumen Secundum
+        </ti:label>
+        <ti:description xml:lang="eng">
+            Agathemerus, creator; Müller, Karl, of Paris, editor
+        </ti:description>
+    </ti:translation>
+    <ti:edition urn="urn:cts:greekLit:tlg0090.tlg001.opp-grc1" workUrn="urn:cts:greekLit:tlg0090.tlg001">
+        <ti:label xml:lang="eng">
```
Comment (lcerrato) [inline data/tlg0090/tlg001/__cts__.xml]: eng --> lat

File: data/tlg0090/tlg001/__cts__.xml [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+    <ti:title xml:lang="grc">Geographiae informatio</ti:title>
+    <ti:translation xml:lang="la" urn="urn:cts:greekLit:tlg0090.tlg001.opp-lat1" workUrn="urn:cts:greekLit:tlg0090.tlg001">
+        <ti:label xml:lang="eng">
+            Geographiae informatio, Geographi graeci minores Volumen Secundum
+        </ti:label>
+        <ti:description xml:lang="eng">
+            Agathemerus, creator; Müller, Karl, of Paris, editor
+        </ti:description>
+    </ti:translation>
+    <ti:edition urn="urn:cts:greekLit:tlg0090.tlg001.opp-grc1" workUrn="urn:cts:greekLit:tlg0090.tlg001">
```
Comment (lcerrato) [inline data/tlg0090/tlg001/__cts__.xml]: edition may require xml:lang as well

File: data/tlg0090/tlg001/tlg0090.tlg001.opp-grc1.xml [hunk 10]
```diff
@@ -0,0 +1,451 @@
...
+                <respStmt>
+                    <persName xml:id="MB">Monica Berti</persName>
+                    <resp>Supervisor</resp>
+                </respStmt>
+                <respStmt>
+                    <persName xml:id="GC">Gregory Crane</persName>
+                    <resp>Supervisor</resp>
+                </respStmt>
+                <respStmt>
+                    <persName> Pasquale Massimo Pinto</persName>
```
Comment (lcerrato) [inline data/tlg0090/tlg001/tlg0090.tlg001.opp-grc1.xml]: remove extra space

File: data/tlg0090/tlg001/tlg0090.tlg001.opp-lat1.xml [hunk 11]
```diff
@@ -0,0 +1,447 @@
...
+                <respStmt>
+                    <persName xml:id="MB">Monica Berti</persName>
+                    <resp>Supervisor</resp>
+                </respStmt>
+                <respStmt>
+                    <persName xml:id="GC">Gregory Crane</persName>
+                    <resp>Supervisor</resp>
+                </respStmt>
+                <respStmt>
+                    <persName> Pasquale Massimo Pinto</persName>
```
Comment (lcerrato) [inline data/tlg0090/tlg001/tlg0090.tlg001.opp-lat1.xml]: remove extra space

**Comment by lcerrato** [comment]
+1 on all of @PonteIneptique suggestions. otherwise looks good!

**Comment by ChiaraPalladino** [comment]
Thanks Lisa. I'm correcting it today. Should I commit directly to the master branch or send another pull request?

**Comment by PonteIneptique** [comment]
Always Pull Request.

**Pull request merged**